### DISCUSSION TITLE
Cannot clone Alpine on Windows because of filenames

### DISCUSSION THREAD

**Discussion by RubenBimmel** [alpinejs/alpine#1565] [7. Bugs (archived)] [answered]
Cloning or fetching the main branch will fail on Windows based filesystems because these two files contain colons in their filenames:

tests/cypress/integration/directives/x-bind:class.spec.js
tests/cypress/integration/directives/x-bind:style.spec.js

Could someone rename these files?

**Comment by calebporzio** [comment, marked as answer]
Good call, thanks. Done! https://github.com/alpinejs/alpine/commit/d41aecd883615b6241be2cb22f075463ddb313df

**Reply by RubenBimmel** [reply]
Thanks! 🎉
